posthog/api/plugin.py

```diff
@@ -310,18 +310,21 @@ def global_plugins(self, request: request.Request, **kwargs):
     @action(methods=["PATCH"], detail=False)
     def rearrange(self, request: request.Request, **kwargs):
         if not can_configure_plugins_via_api(self.team.organization_id):
-            return Response([])
+            return Response(status=403)
```

Comment:
I may totally be in the wrong, but if we're not returning anything here, `raise PermissionDenied` may work better (error message and all).

Reply:
I added a `raise ValidationError("Plugin configuration via the web is disabled!")`. Not sure if that's the right error, but this same line was used a few lines higher in the code, and all errors dispatched here are those. Maybe something to check another time. I'll merge it now though if/when the tests pass.